extensions/arc/deployment/src/main/java/io/quarkus/arc/deployment/configproperties/ConfigPropertiesBuildStep.java

```diff
@@ -1,5 +1,7 @@
 package io.quarkus.arc.deployment.configproperties;
 
+import static io.quarkus.arc.deployment.configproperties.InterfaceConfigPropertiesUtil.*;
```

Comment:
I thought that we should not use wildcard in imports, or?

Reply:
Ah darn... I'll fix it